syntax = "proto3";

package get_item;

option go_package = "./proto"; 

service GetClientService {
    rpc GetClient(GetClientRequest) returns (GetClientResponse) {}
}

message GetClientRequest {
    string id = 1;
}

message GetClientResponse {
    string id = 1;
    string first_name = 2;
    string second_name = 3;
    string phone = 4;
}
